app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,32 @@
+@import "foundation_and_overrides";
+@import "card.scss";
+@import url(https://fonts.googleapis.com/css?family=Quicksand|Raleway);
+
+body {
+  background-image: url("background-photo-stars.jpg");
```

Comment:
Properties should be ordered background-attachment, background-image, background-position, background-repeat, background-size